### PR TITLE
Folder Refactoring, Fragment Logic implementation

### PR DESCRIPTION
**Pull request by CebotariM**
Refactored folder structure, added Fragment MVP logic

### CODE REVIEW

File: app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.example.nvdovin.weatherapp.presentation.main.weather.detail;
+
+public interface DetailsView {
+
```
Comment (victorjimbei) [inline app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java]: I don't see any reason to declare an interface without any method.

File: app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/adapter/TemperatureRecyclerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    @Override
+    public void onBindViewHolder(CustomViewHolder holder, int position) {
+        WeatherData weatherData = weatherDataList.get(position);
+
+        String temperature = String.valueOf(TemperatureConvertor
+                .fromId(sharedPrefs.getTempScaleFromPrefs())
+                .convertToTemperature(weatherData.getTemp().intValue()));
+        holder.temperature.setText(temperature);
+        String date = new SimpleDateFormat("EEE hh.mm").format(weatherData.getDt()*1000);
```
Comment (victorjimbei) [inline app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/adapter/TemperatureRecyclerAdapter.java]: Date format, and millisecond coefficient shouldn't be hardcoded here.

File: app/src/main/res/layout/city_details_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/city_details_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:textSize="73sp"/>
```
Comment (victorjimbei) [inline app/src/main/res/layout/city_details_fragment.xml]: please use dimens

File: app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java (hunk 1 above)
Comment (CebotariM) [reply app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java]: It is not used now, but methods will be added when i will implement US. Delete it anyway?

Comment (victorjimbei) [reply app/src/main/java/com/example/nvdovin/weatherapp/presentation/main/weather/detail/DetailsView.java]: You can leave it for now.

**Pull request closed without merging**